addons/web/static/src/views/pivot/pivot_renderer.xml

```diff
@@ -77,14 +79,22 @@
                     t-on-add-custom-group-by="onAddCustomGroupBy(isXAxis ? 'col' : 'row')"
                 />
             </t>
+            <t t-if="l10n.direction === 'ltr'">
+                <i t-if="cell.isLeaf" class="oi oi-add--thick mr-1"/>
+                <i t-else="!cell.isLeaf" class="oi oi-subtract--thick mr-1"/>
+            </t>
             <span t-esc="cell.title"/>
+            <t t-if="l10n.direction === 'rtl'">
```

Comment:
After trying it the result is not the same. The position of the icon in the DOM seems to be important to have the right result. 
![demo_force_ltr](https://user-images.githubusercontent.com/80679690/149779035-bfd5547f-0d95-4b85-bbc5-eb6d0dfcbe5d.jpg)

Reply:
OK, thanks for checking.

To simplify this block of code, could you apply the condition on `<span t-esc="cell.title"/>` rather than doing that on the icons ?
eg
```xml
<span t-if="l10n.direction === 'rtl'" t-esc="cell.title"/>
<i t-attf-class="oi oi-{{ cell.isLeaf ? 'add' : 'subtract' }}--thick mr-1"/>
<span t-if="l10n.direction === 'ltr'" t-esc="cell.title"/>
```